test/checkpoint/workers.c

```diff
@@ -239,56 +239,52 @@ real_worker(void)
             (void)log_print_err("session.open_cursor", ret, 1);
             goto err;
         }
-    has_cursors = true;
 
-    if (g.use_timestamps)
-        begin_cfg = "read_timestamp=1,roundup_timestamps=(read=true)";
-    else
-        begin_cfg = NULL;
+    if ((ret = session->begin_transaction(session, NULL)) != 0) {
```

Comment:
Not sure what you are doing in this function.

Reply:
I just refactored the whole thing really. I removed the read_timestamp of 1 as that seemed unnecessary from a testing perspective and I also moved around the starting of the transaction to guarantee that all times we have g.use_timestamp = true we actually specify a commit timestamp on the transaction prior to committing it which wasn't true previously.